src/layers/legacy/medCoreLegacy/gui/viewContainers/medViewContainer.cpp

```diff
@@ -514,9 +515,11 @@ void medViewContainer::highlight(QString color)
             d->view->mainWindow()->updateGeometry();
             d->view->mainWindow()->update();
         }
-
```

Comment:
T'es sûr qu'il faut pas le faire dans tout les cas ?

Reply:
i am going to test again if you want

Reply:
i do not see any differences. In medVtkView we always create mainWindow, and viewWidget is applied on it (`d->mainWindow->setCentralWidget(d->viewWidget);`). I think if you update mainWindow, you update viewWidget. However, if mainWindow is not set (in a new type of view for instance), it would be important to update the viewWidget, in a `else` like here.

Reply:
ok